andela/ah-backend-realers
# #166187848 Configuring DB ENGINE

#### What does this PR do?

Configures the project to use postgreSQL as  DB ENGINE instead of using the default engine.

#### Description of task completed

Refactoring DATABASES dictionary in settings.py file under the default key to having PostgreSQL settings

#### How should this be manually tested
* Make sure you have [PostgreSQL](https://www.postgresql.org/download/macosx/) installed on your local machine
* Open a [PostgreSQL](https://www.postgresql.org/download/macosx/) terminal using `$ psql` on your local machine
* Do a `git fetch origin ch-db_connection-166187848`
* Checkout into `ch-db_connection-166187848`
* Install all the dependencies using `pip install -r requirements.txt`
* Create a `.env` file.
* Inside the .env file insert the code below
`export DB_NAME='authors'`
`export DB_USER=''`
`export DB_PASS=''`
`export DB_HOST='localhost'`
`export DB_PORT='5432'`
`echo 'env variables set' `
* Then `source .env` in your terminal
* Then migrate using `$ python manage.py migrate`

[mantains #166187848](https://www.pivotaltracker.com/n/projects/2350340/stories/166187848)
-
